sources/packages/backend/apps/api/src/route-controllers/report/report.controller.service.ts

```diff
@@ -50,6 +50,9 @@ export class ReportControllerService {
         "Not able to export report due to an invalid request.",
       );
     }
+    if (submissionResult.data.data.params["institution"] === "") {
```

Comment:
Can we please add a comment as below?
```
In case the `institution` is present as optional in the submission it will be sent as an empty string (in case it is not provided) or as a number (in case one institution was selected). To ensure the dynamic parameter will always be sent with the same type, the default 0 is used.
```